src/databricks/labs/lsql/dashboards.py

```diff
@@ -668,7 +706,7 @@ def _get_filters_layouts(self) -> Iterable[Layout]:
             layout = Layout(widget=widget, position=position)
             yield layout
 
-    def get_layouts(self) -> Iterable[Layout]:
+    def get_layouts(self, datasets: list[Dataset]) -> Iterable[Layout]:
         """Get the layout(s) reflecting this tile in the dashboard."""
         yield from self._get_query_layouts()
```

Comment:
Signal the datasets is not used, like you did before:

```suggestion
        _ = datasets
        yield from self._get_query_layouts()
```